lib/index.js

```diff
@@ -79,8 +116,7 @@ export default class HyperList {
         return
       }
 
-      const diff = lastRepaint ? scrollTop - lastRepaint : 0
-      if (!lastRepaint || diff < 0 || diff > this._averageHeight) {
+      if (!lastRepaint || Math.abs(scrollTop - lastRepaint) > this._averageHeight) {
```

Comment:
Taken from [6ef597a7af3970f892f8457d5ac5292fdabc1589](https://github.com/tbranyen/hyperlist/commit/6ef597a7af3970f892f8457d5ac5292fdabc1589), but yes, it shouldn't be in the commit